src/main.rs

```diff
@@ -98,15 +98,31 @@ impl EventHandler for Handler {
             }
         } else if WEATHER_RE.is_match(&lower) {
             let typing = msg.channel_id.start_typing(&ctx.http);
-            let address = &lower[14..];
+            let mut units = String::from("standard");
+            let mut offset = 14; // base length of "puppy weather "
+            lazy_static! {
```

Comment:
what if there's a place that starts with "imperial" like `puppy weather imperial treasure`, which is the name of a [chinese restaurant in palo alto](https://maps.app.goo.gl/dx2nBLZK2anSx8nK6)

Reply:
maybe we can just call it `celsius`, `fahrenheit`, and `kelvin` instead of "standard", "imperial", and "metric" --- after all, both kelvin and celsius are metric units and it is relatively less likely for a place name to start with those three words

Reply:
fixed as suggested